packages/ui-system/src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from "./Loader.module.scss";
+
```

Comment:
**praise**: Great job on making the `Loader` component flexible by allowing children to be passed in.